healthcheck.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# First argument is the maximum age in seconds (default: 60)
+maxage=${1:-60}
+
+# Get the age of the last successful metrics refresh in seconds
+age=$(curl -s http://localhost:9040/metrics | grep 'homematic_refresh_age{' | cut -d ' ' -f2 | cut -d '.' -f1)
```

Comment:
no curl in the docker image yet - and the scripts lacks an execute bit.

Reply:
Oh wow, I should have checked the healthcheck logs!

I didn't need to change the file permissions though. It's now doing 0-exit on my deployment..

Reply:
your healthcheck command included the bash-invocation and only gave the script as argument, that is why you didn't nee the bit. But it would make sense to do so to be able to call it natively.

Reply:
Please feel free to suggest a change or make a commit. I don't know what exactly you'd like to see here

Reply:
See https://github.com/sfudeus/homematic_exporter/pull/58/commits/5f580ed66f54142aa6f6574091ecdfeb648e8dbe, it is just about the permissions bits on the healthcheck.sh. I can add that later myself as well, I cannot push to your branch. 

The most recent docker build contains that already and looks good so far in my cluster. I did not set a liveness check using the healthcheck yet.

Reply:
The Linux file permissions at the git level? I didn't know these were copied into the docker image.